src/components/filters/MeasureValueFilter/Dropdown.tsx

```diff
@@ -21,6 +21,8 @@ export interface IDropdownOwnProps {
     locale?: string;
     anchorEl: EventTarget | string;
     separators?: ISeparators;
+    displayTreatNullAsZeroOption?: boolean;
```

Comment:
please provide default values explicitly for this react component. See src/components/filters/AttributeFilter/AttributeFilter.tsx `public static defaultProps: Partial<IAttributeFilterProps> = {`